metadata-ingestion-modules/airflow-plugin/setup.py

```diff
@@ -47,19 +52,18 @@ def get_long_description():
 base_dev_requirements = {
     *base_requirements,
     *mypy_stubs,
-    "black>=21.12b0",
+    "black==22.12.0",
     "coverage>=5.1",
     "flake8>=3.8.3",
     "flake8-tidy-imports>=4.3.0",
     "isort>=5.7.0",
-    "mypy>=0.920",
+    "mypy==1.0.0",
```

Comment:
can we work with a newer version of mypy?

Reply:
I had just picked this from main ingestion setup.py. Let me try to make airflow-plugin work with higher mypy version.

Reply:
setting it to mypy>=1.4.0, as python 3.7 is not supported for mypy>=1.5.0